versioned_docs/version-22.04/releases/centreon-core.md

```diff
@@ -17,6 +17,37 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 22.04.6
+
```

Comment:
[Administration] Access group fields are not mandatory for OpenId Connect configuration